server/app/controllers/dev/seeding/QuestionDefinitions.java

```diff
@@ -0,0 +1,208 @@
+import services.question.types.QuestionDefinitionConfig;
+import services.question.types.StaticContentQuestionDefinition;
+import services.question.types.TextQuestionDefinition;
+
+public final class QuestionDefinitions {
+
+  public static final AddressQuestionDefinition ADDRESS_QUESTION_DEFINITION =
+      new AddressQuestionDefinition(
+          QuestionDefinitionConfig.builder()
+              .setName("Sample Address Question")
+              .setDescription("description")
+              .setQuestionText(LocalizedStrings.withDefaultValue("What is your address?"))
+              .setQuestionHelpText(LocalizedStrings.withDefaultValue("help text"))
+              .build());
+  private static final QuestionDefinitionConfig CHECKBOX_QUESTION_DEFINITION_CONFIG =
+      QuestionDefinitionConfig.builder()
+          .setName("checkbox")
+          .setDescription("description")
+          .setQuestionText(
+              LocalizedStrings.withDefaultValue(
+                  "Which of the following kitchen instruments do you own?"))
+          .setQuestionHelpText(LocalizedStrings.withDefaultValue("help text"))
+          .build();
+  private static final ImmutableList<QuestionOption> CHECKBOX_QUESTION_OPTIONS =
+      ImmutableList.of(
+          QuestionOption.create(1L, 1L, LocalizedStrings.withDefaultValue("toaster")),
+          QuestionOption.create(2L, 2L, LocalizedStrings.withDefaultValue("pepper grinder")),
+          QuestionOption.create(3L, 3L, LocalizedStrings.withDefaultValue("garlic press")));
+
+  public static final MultiOptionQuestionDefinition CHECKBOX_QUESTION_DEFINITION =
+      new MultiOptionQuestionDefinition(
+          CHECKBOX_QUESTION_DEFINITION_CONFIG,
+          CHECKBOX_QUESTION_OPTIONS,
+          MultiOptionQuestionType.CHECKBOX);
+
+  public static final CurrencyQuestionDefinition CURRENCY_QUESTION_DEFINITION =
+      new CurrencyQuestionDefinition(
+          QuestionDefinitionConfig.builder()
+              .setName("Sample Currency Question")
+              .setDescription("description")
+              .setQuestionText(
+                  LocalizedStrings.withDefaultValue("How much should a scoop of ice cream cost?"))
+              .setQuestionHelpText(LocalizedStrings.withDefaultValue("help text"))
+              .build());
+
+  private static final QuestionDefinitionConfig.Builder DATE_QUESTION_DEFINITION_CONFIG_BUILDER =
+      QuestionDefinitionConfig.builder()
+          .setName("Sample Enumerator Date Question")
+          .setDescription("description")
+          .setQuestionText(LocalizedStrings.withDefaultValue("When is $this's birthday?"))
+          .setQuestionHelpText(LocalizedStrings.withDefaultValue("help text for $this's birthday"));
+
+  public static DateQuestionDefinition dateQuestionDefinition(long enumeratorId) {
+    return new DateQuestionDefinition(
+        DATE_QUESTION_DEFINITION_CONFIG_BUILDER.setEnumeratorId(enumeratorId).build());
```

Comment:
Maybe just inline this. Or consider using dateQuestionDefinition("Sample Enumerator Date Question", "When is $this's birthday?);

Reply:
Inlined.